src/Server.hpp

```diff
@@ -66,7 +68,7 @@ class Server
 	void listen();
 	/* for startPollEventLoop */
 	void addServerSocketPollFdToVectors();
-	void acceptNewConnection();
+	void acceptNewConnection(Connection &conn);
```

Comment:
What is this change ?

Reply:
Ah. Before we were taking the _serverFd (as we had only one). This is the preparation for the multi 'server' thing. We are taking the Server Connection Object which has the POLLIN, to pass the information of this server connection to the client connection we are going to create. Maybe it would have been better to call the parameter Connection &ServerConn

Reply:
Alright, thanks for the explanation